src/utils/event.js

```diff
@@ -62,6 +30,9 @@ export const getOffersPrice = (event) => {
 };
 
 export const humanizeDate = (dueDate) => {
+  if (!dueDate) {
+    return ``;
+  }
   return dueDate.toLocaleString(`en-US`, {day: `numeric`, month: `long`}) || ``;
```

Comment:
```suggestion
  return dueDate && dueDate.toLocaleString(`en-US`, {day: `numeric`, month: `long`}) || ``;
```

Reply:
или, если опшнал-чейнинг включен в бабеле, то вообще вот так можно
```suggestion
  return dueDate?.toLocaleString(`en-US`, {day: `numeric`, month: `long`}) || ``;
```

Reply:
линт ругается, а так бы я уже давно его использовала)